security/pfSense-pkg-HoneyTrap/Makefile

```diff
@@ -0,0 +1,53 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-HoneyTrap
+PORTVERSION=	0.1
+CATEGORIES=	security
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	remco.verhoef@dutchsec.com
+COMMENT=	PfSense package HoneyTrap
+
+LICENSE=	ART20
```

Comment:
All files from this port have header saying they are distributed under Apache 2.0 license, so it should be APACHE20 here